### DISCUSSION TITLE
Glossary

### DISCUSSION THREAD

**Discussion by saraa-costa** [joaomlourenco/novathesis#433] [General]
Hi. I am having troubles with glossary.
I can't understand how this works.

**Comment by joaomlourenco** [comment]
Hello Sara!
Have you seen the examples in the original distribution of the template? What are you not understanding?

**Reply by saraa-costa** [reply]
Hi again!
Yes, I saw the example of the word "computer" but when I click on the word, the meaning doesn't appear, that is, latex sends me to the cover.

**Comment by joaomlourenco** [comment]
If you process the template correctly (the right number of times) it will work as expected.
Please download the PDF from the repository and confirm what I am saying, that the link in the "computer" works as expected.

**Reply by saraa-costa** [reply]
I have already downloaded the PDF from the repository four times and none of them show the glossary.
So I don't understand what " process the template correctly (the right number of times)" means...

**Reply by joaomlourenco** [reply]
`So I don't understand what " process the template correctly (the right number of times)" means...`
Definitely…  😊
It depends on your working environment…
Please read point 3. in this link“ https://github.com/joaomlourenco/novathesis#getting-started and, maybe, search in google for "Configure Latex Editor EDITOR_NAME" and see how to make it work for your editor.  Then come back with additional difficulties.

**Reply by saraa-costa** [reply]
I'm using overleaf not a Local LaTeX.

**Reply by joaomlourenco** [reply]
You have not said that before…. 😢
Then it should work… you are telling me that, if I download the template from the GitHub repository and upload it to Overleaf, the glossary will not be all right?  Please confirm!
